### PR TITLE
Make verbose flag default to true when a callbackManager is passed in

### LINKED ISSUE #507: Default verbose to true when passing in callbackManager
This isn't an ideal api but that's a different problem

### PR DESCRIPTION
**Pull request by nfcampos**
Closes #507

### CODE REVIEW

File: langchain/src/callbacks/base.ts [hunk 1]
```diff
@@ -350,27 +350,35 @@ export class CallbackManager extends BaseCallbackManager {
 }
 
 export class ConsoleCallbackHandler extends BaseCallbackHandler {
+  async handleLLMStart(llm: { name: string }, prompts: string[]) {
```
Comment (hwchase17) [inline langchain/src/callbacks/base.ts:353]: not sure if intentional or was just for testing, but flagging that this changed

**Review by hwchase17 — APPROVED** [review]

File: langchain/src/callbacks/base.ts (hunk 1 above)
Comment (nfcampos) [reply langchain/src/callbacks/base.ts:353]: Yea that was on purpose, seemed a bit strange that our console callback handler only implements some methods? But can revert if that's on purpose for some reason

Comment (agola11) [reply langchain/src/callbacks/base.ts:353]: I think we'll want to keep as is otherwise there will be too much output during agent runs

**Review by agola11 — COMMENTED** [review]
Looks good, I would just revert the console callback handler to keep the agent output as is

File: langchain/src/callbacks/base.ts (hunk 1 above)
Comment (nfcampos) [reply langchain/src/callbacks/base.ts]: Alright, undid these changes

File: langchain/src/chains/llm_chain.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export interface LLMChainInput extends ChainInputs {
   outputParser?: BaseOutputParser;
 
   /** @ignore */
-  outputKey: string;
+  outputKey?: string;
```
Comment (agola11) [inline langchain/src/chains/llm_chain.ts:29]: why did this change?

**Review by agola11 — APPROVED** [review]

File: langchain/src/chains/llm_chain.ts (hunk 2 above)
Comment (nfcampos) [reply langchain/src/chains/llm_chain.ts:29]: Because I'm now using this type in the constructor, and it's optional there

**Pull request merged**